esti/hooks_test.go

```diff
@@ -7,5 +7,5 @@ import (
 func TestHooksSuccess(t *testing.T) {
 	ctx, _, repo := setupTest(t)
 	defer tearDownTest(repo)
-	HooksSuccessTest(ctx, t, repo)
+	HooksSuccessTest(ctx, t, repo, nil)
```

Comment:
need to create client or there is a default one?

Reply:
defaullt